build-config/Makefile

```diff
@@ -288,10 +289,14 @@ endif
 # Default mirror for packages needed by ONIE
 ONIE_MIRROR	?= http://mirror.opencompute.org/onie
 
-include make/crosstool-ng.make
 include make/kernel-download.make
-include make/uclibc-download.make
-include make/xtools.make
+ifneq ($(XTOOLS_LIBC),glibc)
+  include make/uclibc-download.make
+endif
+ifneq ($(XTOOLS_ENABLE),no)
```

Comment:
I am not a fan of double negatives.  Can this be written as:
```
ifeq ($(XTOOLS_LIBC),uclibc)
```

and

```
ifeq ($(XTOOLS_ENABLE),yes)
```

Also does it make sense to have the XTOOLS_LIBC checkout outside of the XTOOLS_ENABLE check?  Meaning, does the setting of XTOOLS_LIBC matter at all if XTOOLS_ENABLE is disabled?